chart/parser.go

```diff
@@ -92,9 +95,21 @@ func (c *Chart) compile(releaseName, namespace, values string) ([]byte, error) {
 		Namespace: namespace,
 	}
 
+	caps := &chartutil.Capabilities{
+		APIVersions: chartutil.DefaultVersionSet,
+		KubeVersion: &version.Info{
```

Comment:
Any reason you didn't use `chartutil.DefaultKubeVersion` here? I guess it might not be available in the helm version Kubecrt is using?